app/services/samples/transfer_service.rb

```diff
@@ -77,6 +77,7 @@ def transfer(new_project_id, sample_ids) # rubocop:disable Metrics/MethodLength,
 
         @project.namespace.update_metadata_summary_by_sample_transfer(transferred_samples_ids,
                                                                       new_project_id)
+        update_samples_count(transferred_samples_ids.count) if @project.parent.type == 'Group'
```

Comment:
**nitpick (blocking):** If I create a project under my user namespace with a sample, and then transfer that sample to a project that is under a group. The group does not update the sample count and it remains at 0.